service/application/src/main/java/com/solace/maas/ep/event/management/agent/config/SolaceConfiguration.java

```diff
@@ -48,14 +49,19 @@ public SolaceConfiguration(Properties vmrConfig, ArrayList<String> sessionConfig
         this.eventPortalProperties = eventPortalProperties;
     }
 
-    public String getTopicPrefix() {
-
-        if (topicPrefix == null) {
-            topicPrefix = String.format(TOPIC_PREFIX_FORMAT,
-                    eventPortalProperties.getOrganizationId(),
-                    eventPortalProperties.getRuntimeAgentId());
+    public String getTopicPrefix(String orgId) {
+        if (StringUtils.isEmpty(orgId)) {
+            log.debug("Attempted to get topic prefix with empty orgId. Defaulting to application properties org ID {}",
+                    eventPortalProperties.getOrganizationId());
+            orgId = eventPortalProperties.getOrganizationId();
         }
-        return topicPrefix;
+        return String.format(TOPIC_PREFIX_FORMAT,
+                orgId,
+                eventPortalProperties.getRuntimeAgentId());
+    }
+
+    public String getTopicPrefix() {
+        return getTopicPrefix(null);
     }
```

Comment:
There's several places where we call this to do things like set the config push command subscription and set the topic to send responses to. It's used for scan, config push, and heartbeat. To make this PR low touch, I've only altered anything to do with config push. That's the only case that sends in an orgId right now.
Maybe scan would want to use this as well? cc @rudraneel-chakraborty